src/components/faceVerification/standalone/context/FVFlowContext.js

```diff
@@ -18,9 +20,13 @@ export const FVFlowContext = createContext({
   chain: null,
   rdu: null,
   cbu: null,
+  isWebView: false,
+  unsupportedCopyUrl: null,
 })
 
 const FVFlowProvider = props => {
+  const unsupportedCopyUrl = window.location.href
```

Comment:
see if url is available in DeepLinking just so it is more standardized

Reply:
fixed